exercises/practice/grains/src/lib.rs

```diff
@@ -1,7 +1,18 @@
+const SQUARE_OUT_OF_RANGE: &str = "Square must be between 1 and 64";
```

Comment:
So now, if I have to change the size of the board I also need to change this const string.
Also I think having a const string here is kind of hard to read code. Just write the error message in the same line